suites/nautilus/ansible/tier-0_deploy_containerized_ceph/02_ceph_ansible/overrides.yaml

```diff
@@ -0,0 +1,48 @@
+tests:
```

Comment:
Instead of an override, I believe this test is unique ... can we create a fragment?

The override can be `alertmanager_container_image`